packages/core-contracts/contracts/mocks/token/ERC721FailedReceiverMock.sol

```diff
@@ -0,0 +1,15 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../interfaces/IERC721Receiver.sol";
+
+contract ERC721FailedReceiverMock is IERC721Receiver {
```

Comment:
there are a couple of ERC721Receiver mocks to mimic the different possible failures. This one is one that implements the ERC721Receiver interface poorly (wrong return value)